documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md

```diff
@@ -0,0 +1,27 @@
+# API Readiness Checklist
+
+Checklist for Carrier Billing Refund v0.1.0 in r0.3.
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | link |
```

Comment:
Just as a hint, you can use relative links for these links, which are independent of the actual release tag, example:

```suggestion
|  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | [link](/code/API_definitions/carrier_billing_refund.yaml) |
```

Reply:
Thanks for this comment indication